security/cert_manager_operator/index.adoc

```diff
@@ -20,9 +20,16 @@ include::modules/cert-manager-request-methods.adoc[leveloffset=+1]
 //Supported versions
 include::modules/cert-manager-supported-versions.adoc[leveloffset=+1]
 
+//FIPS compliant support
+include::modules/cert-manager-fips-support.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 [id="cert-manager-operator-about_additional-resources"]
 == Additional resources
 
 * link:https://cert-manager.io/docs/[cert-manager project documentation]
+* xref:../../security/container_security/security-compliance.adoc#security-compliance[Understanding compliance]
+* xref:../../installing/installing-fips.adoc#installing-fips-mode_installing-fips[Installing a cluster in FIPS mode]
+* link:https://csrc.nist.gov/Projects/cryptographic-module-validation-program/validated-modules[Cryptographic module validation program]
+* link:https://access.redhat.com/articles/2918071#fips-140-2-and-fips-140-3-2[Compliance activities and government standards]
```

Comment:
These two could be moved up into the text, as they are external links rather than xrefs